src/frequenz/client/dispatch/types.py

```diff
@@ -33,10 +33,32 @@
 from google.protobuf.struct_pb2 import Struct
 
 from frequenz.client.base.conversion import to_datetime, to_timestamp
-from frequenz.client.common.microgrid.components import ComponentCategory
+from frequenz.client.common.microgrid.components import (
+    ComponentCategory,
+)
+from frequenz.client.common.microgrid.electrical_components import (
+    ElectricalComponentCategory,
+)
+from frequenz.client.common.streaming import Event
 
 from .recurrence import Frequency, RecurrenceRule, Weekday
 
+# Re-export Event for backwards compatibility
+__all__ = [
+    "Event",
```

Comment:
To mark the import location as deprecated you could also instead of re-export it here, add a class like:

```py
@deprecated("Use from frequenz.client.common.streaming.Event instead")
class Event(streaming.Event): pass
```

Reply:
src/frequenz/client/dispatch/test/_service.py:202: error: Argument "event" to "DispatchEvent" has incompatible type "frequenz.client.common.streaming.Event"; expected "frequenz.client.dispatch.types.Event"  [arg-type]

That doesn't seem to be equivalent at least